docs/ui/prepare-migration/monitor.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: Monitoring
+parent: Prepare Migration Page
+grand_parent: SMT UI
+nav_order: 4
+---
+
+# Monitoring
+{: .no_toc }
+
+HarbourBridge tracks the updates for schema migration, data migration and foreign key update. This information is polled from the backend at **5 sec** intervals and presented in the UI with separate progress bars. In case of an error during migration, the state of these progress bars is marked as canceled. After the successful completion of migration the generated resources are presented to the users for utilization.
+
+In case of minimal downtime migration with GoogleSQL dialect, the foreign keys are applied along with the schema migration.
```

Comment:
lets specify that for minimal downtime migrations the monitoring currently is only till job creation

Reply:
Done